Packs/ServiceNow/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,40 @@
+
+#### Integrations
+##### ServiceNow v2
+- Added the ticket mirroring feature.
+
+#### Incident Fields
+- **ServiceNow Urgency**
+- **ServiceNow Category**
+- **ServiceNow Caller**
+- **ServiceNow Assignment Group**
+- **ServiceNow Assigned To**
+- **ServiceNow State**
+- **ServiceNow Severity**
+- **ServiceNow Resolved Time**
+- **ServiceNow Resolution Notes**
+- **ServiceNow Priority**
+- **ServiceNow Opened Date**
+- **ServiceNow Notify**
+- **ServiceNow Impact**
+- **ServiceNow Escalation**
+- **ServiceNow Due Date**
+- **ServiceNow Description**
+- **ServiceNow Closed Date**
+- **ServiceNow Ticket Number**
+
+#### Layouts
+- **ServiceNow Ticket**
+
+#### Classifiers
+##### New: ServiceNow Classifier
+- Classifies ServiceNow tickets.
+
+##### New: ServiceNow - Outgoing Mapper
+- Maps outgoing ServiceNow incident Fields.
+
+##### New: ServiceNow - Incoming Mapper
+- Maps incoming ServiceNow incidents fields.
+
+##### New: ServiceNow
```

Comment:
what is that ?

Reply:
Was generated automaticlly with the sdk command. Will remove